src/test/java/no/nav/familie/integrasjoner/oppgave/OppgaveServiceTest.kt

```diff
@@ -113,4 +114,93 @@ internal class OppgaveServiceTest {
         assertThat(oppgaveSlot.captured.aktoerId).isEqualTo(aktørid)
         assertThat(oppgaveSlot.captured.personident).isNull()
     }
+
+    @Test
+    fun `tilordneEnhetOgNullstillTilordnetRessurs skal fjerne mappe fra oppgave dersom det settes til true`() {
```

Comment:
Her kunne du laget en `@nested inner class TilordneEnhetOgNullstillTilordnetRessursTest` for å bedre gruppere testene